### PR TITLE
Add toAPIVersion method to APIService

### PR DESCRIPTION
**Pull request by benjaminapetersen**

### CODE REVIEW

File: dist/origin-web-common-services.js [hunk 1]
```diff
@@ -260,6 +260,15 @@ angular.module('openshiftCommonServices')
     return new ResourceGroupVersion(resource, group, version);
   };
 
+  // toGroupVersion() returns a group/version string
+  //
+  var toGroupVersion = function(resourceGroupVersion) {
```
Comment (benjaminapetersen) [inline dist/origin-web-common-services.js]: I'm trying to decide if this name is ok, given that the other fns with similar names return objects.

Comment (jwforres) [reply dist/origin-web-common-services.js]: the appropriate name is toAPIVersion I think, since that is the field in the API objects that it refers to

Comment (spadgett) [reply dist/origin-web-common-services.js]: +1 for `toAPIVersion`

Comment (benjaminapetersen) [reply dist/origin-web-common-services.js]: perfect, updating.

**Comment by benjaminapetersen** [comment]
updated

**Pull request merged**